spec/client_spec.rb

```diff
@@ -1,10 +1,42 @@
+require 'config_helper'
 
 describe SwiftypeAppSearch::Client do
   let(:engine_name) { "ruby-client-test-#{Time.now.to_i}" }
 
   include_context "App Search Credentials"
   let(:client) { SwiftypeAppSearch::Client.new(client_options) }
 
+  before(:all) do
+    # Bootstraps a static engine for 'read-only' options that require indexing
+    # across the test suite
+    @static_engine_name = "ruby-client-test-static-#{Time.now.to_i}"
```

Comment:
I mentioned that in the PR description. 

These are integration tests against the live API. Most tests create a new client and engine in a `before` and `after`.

Some operations, like search, require a document to be indexed. Since indexing happens asynchronously, and can take up to a minute or more to index, we create a single, "static" index and client in `before(:all)` for use all throughout the test suite.

So client and engine are ephemeral for each test, and static is available throughout the entire test suite for operations that require indexed documents.